pkgs/top-level/all-packages.nix

```
@@ -5738,6 +5738,8 @@ in
 
   otfcc = callPackage ../tools/misc/otfcc { };
 
+  otpgen = libsForQt5.callPackage ../tools/security/otpgen {};
```

And here we usually include space :wink: 

```suggestion
  otpgen = libsForQt5.callPackage ../tools/security/otpgen { };
```